### ISSUE TITLE
test-utils code completion is not working

### ISSUE THREAD

**Issue by ahmehri** [commercetools/merchant-center-application-kit#1076]
**Describe the bug**
`@testing-library/react` autocompletion is working only when importing directly from `@testing-library/react` package.

**To Reproduce**

1. Use the following code
```js
import { render } from '@testing-library/react';
import {
  renderApp,
  experimentalRenderAppWithRedux,
} from '@commercetools-frontend/application-shell/test-utils';
import { ApplicationProducts } from '../entry-point';

it('test', () => {
  const rendered1 = renderApp(<ApplicationProducts />);
  const rendered2 = experimentalRenderAppWithRedux(<ApplicationProducts />);
  const rendered3 = render(<ApplicationProducts />);
});
```
2. check auto completion with each variable

**Expected behavior**
auto completion should works also with `rendered1` and `rendered2`.

**Screenshots**
![Kapture 2019-09-27 at 12 06 34](https://user-images.githubusercontent.com/3668245/65761582-56eb2e80-e11f-11e9-8027-bf841120fb58.gif)

**Comment by tdeekens** [comment]
Thanks for reporting. Did we maybe fail to export the types from our own test-utils or they‘re not copied to our dist and we should do so upon build.

**Comment by emmenko** [comment]
I think it might be related to the fact that our test-utils do not have type declarations yet. Once we migrated them to TS, things should properly work I presume.

**Comment by emmenko** [comment]
This will be fixed with #1069 and included in v16 #1248
